### PR TITLE
feat:add skip to content link

### PR DESCRIPTION
**Pull request by ketanbaitule**
## What does this PR do?

Add Skip To Content Link

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

Solves #278 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by ketanbaitule** [comment]
[Skip Content Demo](https://github.com/appwrite/website/assets/111437670/6037306f-aebc-4a5e-923e-48f134364416)

**Comment by ketanbaitule** [comment]
Fix #278

File: src/routes/docs/quick-starts/+page.svelte [hunk 1]
```diff
@@ -127,7 +127,7 @@
 	<meta name="twitter:card" content="summary_large_image" />
 </svelte:head>
 
-<main class="aw-main-section">
+<main class="aw-main-section" id="main">
```
Comment (TGlide) [inline src/routes/docs/quick-starts/+page.svelte:130]: Adding it to pages alone won't be enough, we should add them to all relevant layouts

File: src/lib/layouts/Docs.svelte [hunk 2]
```diff
@@ -59,7 +59,20 @@
...
 </script>
-
+<style>
+    .skip-to-content-link {
+        z-index: 1000;
+        padding: 8px;
+        position: absolute;
+        top: 0;
+        -webkit-transform: translateY(-999px);
+        transform: translateY(-999px);
```
Comment (TGlide) [inline src/lib/layouts/Docs.svelte:69]: Wouldn't it be better to use opacity instead of translations?

**Review by TGlide — COMMENTED** [review]
Thank you for the PR! I left some comments, and also let our design team know about this so they can review it.

**Comment by ariascaio** [comment]
@ketanbaitule I'm a member of the Design team at Appwrite :)
I did a few adjustments to the Skip to Content banner for you to implement, you can [check it on Figma](https://www.figma.com/file/i5gnXFw6WWXQ0VWT6ER2j9/Appwrite---Skip-to-Content-banner?type=design&node-id=0%3A1&mode=design&t=bfX9UEfYM6jrlETN-1)

**Review by TGlide — APPROVED** [review]

**Comment by TGlide** [comment]
> @TGlide Hi I have not completed it due to my exams. I guess you have mistakenly merged it

I've made some commits on top of your work before merging it 🙂

**Pull request merged**